packages/packages/documents/src/hooks/__tests__/use-current-document.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { Slice } from '../../types';
+import { PropsWithChildren } from 'react';
+import { createSlice } from '../../store';
+import { renderHook } from '@testing-library/react-hooks';
+import { useCurrentDocument } from '../use-current-document';
+import { createStore, deleteStore, dispatch, SliceState, Store, StoreProvider } from '@elementor/store';
+
+describe( '@elementor/documents/hooks/use-current-document', () => {
+	let store: Store<SliceState<Slice>>;
+	let slice: Slice;
+
+	beforeEach( () => {
+		slice = createSlice();
+		store = createStore();
+	} );
+
+	afterEach( () => {
+		deleteStore();
+	} );
+
+	it( 'should return the current document', () => {
+		// Arrange.
+		const mockDocument = {
```

Comment:
Don't you have a dedicated function that generates the mock? if not, maybe create one and use it everywhere?

Reply:
not the same document